src/components/app/dtsiClientPersonDataTable/common/useTableFilters.ts

```diff
@@ -64,38 +67,37 @@ export function useSortingFilter(
   return [hasHydrated ? sortedValue : [], handleSortingValue]
 }
 
+const roleKey = PERSON_TABLE_COLUMNS_IDS.ROLE
+const partyKey = PERSON_TABLE_COLUMNS_IDS.PARTY
+const stateKey = PERSON_TABLE_COLUMNS_IDS.STATE
+const stanceKey = PERSON_TABLE_COLUMNS_IDS.STANCE
+
+const defaultFiltersValue = 'All'
```

Comment:
Just for consistency with other files

```suggestion
const ROLE_KEY = PERSON_TABLE_COLUMNS_IDS.ROLE
const PARTY_KEY = PERSON_TABLE_COLUMNS_IDS.PARTY
const STATE_KEY = PERSON_TABLE_COLUMNS_IDS.STATE
const STANCE_KEY = PERSON_TABLE_COLUMNS_IDS.STANCE

const DEFAULT_FILTERS_VALUE = 'All'
```